src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -284,6 +319,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     // Function Description:
     // - Updates the currently modifiable color table based on the given current color scheme.
+    //   There are 7 non-bright colours and 7 bright colours.  
```

Comment:
This is false. we have 8+8 colors. But the list index goes from 0-7. If you cont ftom 0-7 you count 8 times.